app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -40,16 +40,15 @@ Reference links:
 https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/installation_guide/s1-kickstart2-options
 https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/sect-kickstart-syntax
 https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_installation/kickstart-commands-and-options-reference_installing-rhel-as-an-experienced-user
-%>
-
-# This kickstart file was rendered from the Foreman provisioning template "<%= @template_name %>".
-
+-%>
 <%
   rhel_compatible = @host.operatingsystem.family == 'Redhat' && @host.operatingsystem.name != 'Fedora'
   is_fedora = @host.operatingsystem.name == 'Fedora'
   os_major = @host.operatingsystem.major.to_i
   os_minor = @host.operatingsystem.minor.to_i
   realm_compatible = (@host.operatingsystem.name == 'Fedora' && os_major >= 20) || (rhel_compatible && os_major >= 7)
+  organization = @host.info['parameters']['organization']
+  location = @host.info['parameters']['location']
```

Comment:
Due to facets, info can actually return a hash without any parameters which would lead to nil exception. Please do this:

```suggestion
  location = @host.info.fetch('parameters', 'location')
```

The same for organization. This method is in safemode too.